protoc-gen-go/generator/generator.go

```diff
@@ -92,6 +92,12 @@ func RegisterPlugin(p Plugin) {
 	plugins = append(plugins, p)
 }
 
+// An ImportPath is the import path of a Go package. e.g., "github.com/golang/protobuf".
+type ImportPath string
+
+// A PackageName is the name of a Go package. e.g., "protobuf".
+type PackageName string
```

Comment:
Ditto, rename as `GoPackageName` to disambiguate from proto packages?

Reply:
done